src/common/cpu.c

```diff
@@ -175,11 +175,11 @@ char* get_str_peak_performance(int64_t flops) {
   str = ecalloc(max_size, sizeof(char));
 
   if(flopsd >= (double)1000000000000.0)
-    snprintf(str, max_size, "%.2f TFLOP/s", flopsd/1000000000000);
+    snprintf(str, max_size, "%.2f TFLOP/s", (float)flopsd/1000000000000);
```

Comment:
Remove all unnecesary the the float conversion. We'll address the peak performance issue later.